pkg/suggestion/v1alpha2/bayesianoptimization/src/algorithm_manager.py

```diff
@@ -0,0 +1,201 @@
+""" module for algorithm manager """
+import numpy as np
+
+from pkg.api.v1alpha2.python import api_pb2
+
+from .utils import get_logger
+
+
+def deal_with_discrete(feasible_values, current_value):
```

Comment:
This contains utils that are already in parsing_utils. We can merge it later.

Reply:
SGTM.